benchmark/citation/train_eval.py

```diff
@@ -82,12 +83,56 @@ def run(dataset, model, runs, epochs, lr, weight_decay, early_stopping,
         val_losses.append(best_val_loss)
         accs.append(test_acc)
         durations.append(t_end - t_start)
-
     loss, acc, duration = tensor(val_losses), tensor(accs), tensor(durations)
 
     print(f'Val Loss: {float(loss.mean()):.4f}, '
           f'Test Accuracy: {float(acc.mean()):.3f} ± {float(acc.std()):.3f}, '
-          f'Duration: {float(duration.mean()):.3f}')
+          f'Duration: {float(duration.mean()):.3f}s')
+
+
+def run_inference(dataset, model, runs, epochs, profiling, permute_masks=None,
+                  logger=None):
+    for i in range(runs):
```

Comment:
Looks like we never really iterate over `runs` - how about we simply remove `runs` from the argument list?

Reply:
Sure. I will update this.

Reply:
Sorry, here is the code indent issue when I copy the original code. `runs` will be used.